File: assgn2/Subset.java
public class Subset {
    public static void main(String[] args) {
        int k = Integer.parseInt(args[0]);
        RandomizedQueue<String> rq = new RandomizedQueue<String>();
        for (int i = 0; i < k; i++){
            String s = StdIn.readString();
            rq.enqueue(s);
        }

        for(int i = 0; i < k; i++) {
            StdOut.println(rq.dequeue());
        }
    }
}

File: python/queues/subset.py
#!/usr/bin/python2
from randomized_queue import RandomizedQueue
import sys
k = int(sys.argv[1])
r = sys.stdin.read().split()
rq = RandomizedQueue()
for x in r:
    rq.enqueue(x)

for i in range(k):
    print rq.dequeue()

